components/clarinet-deployments/src/lib.rs

```diff
@@ -296,12 +306,30 @@ fn handle_emulated_contract_call(
     result
 }
 
+// Main function that always includes boot contracts by default
```

Comment:
Remove this comment.
I think the comment only make sense in the context of this PR. But if someone discover the repo and sees this comment, that won't be helpful at all.